### ISSUE TITLE
[editor] chinese input method error

### ISSUE THREAD

**Issue by 502647092** [eclipse-theia/theia#6459]
### Description

at theia when I input at input method then backspace it some thing at editor will clear
![temp](https://user-images.githubusercontent.com/10203793/67758329-73f58300-fa78-11e9-8021-c78bf288c475.gif)

I test at monaco don't have this bug
![temp](https://user-images.githubusercontent.com/10203793/67758147-13664600-fa78-11e9-9bac-4544e264ae33.gif)

### Reproduction Steps
see gif
**OS and Theia version:**
**Diagnostics:**

**Comment by monaka** [comment]
I got the almost same issue on Che-Theia with ChromeOS/Japanese input method.
 
I suspect the root cause is "Current Theia doesn't care about `KeyCode.KEY_IN_COMPOSITION`."
This will be not the issue of Monaco but Theia.
I found the similar issue in VSCode and there looks be fixed by  https://github.com/microsoft/vscode/pull/40546.

I guess there needs some more code in `packages/core/src/browser/widgets/widget.ts` or near.
